test/NuGet.Clients.FuncTests/NuGet.CommandLine.FuncTest/Commands/SearchCommandTests.cs

```diff
@@ -0,0 +1,138 @@
+      ""@type"": ""SearchQueryService/Versioned"",
+      ""comment"": ""Query endpoint of NuGet Search service (primary)""
+    }}
+    ],
+    ""@context"": {{
+    ""@vocab"": ""http://schema.nuget.org/services#"",
+    ""comment"": ""http://www.w3.org/2000/01/rdf-schema#comment""
+  }}
+        }}
+            ";
+
+                server.Get.Add("/v3/index.json", r => index);
+
+
+
+                string queryResult = $@"
+                {{
+                    ""@context"":
+                    {{
+                        ""@vocab"": ""http://schema.nuget.org/schema#"",
+                        ""@base"": ""https://api.nuget.org/v3/registration5-semver1/""
+                    }},
+                    ""totalHits"": 396,
+                    ""data"": [
+                    {{
+                        ""@id"": ""https://api.nuget.org/v3/registration5-semver1/newtonsoft.json/index.json"",
+                        ""@type"": ""Package"",
+                        ""registration"": ""https://api.nuget.org/v3/registration5-semver1/newtonsoft.json/index.json"",
+                        ""id"": ""Newtonsoft.Json"",
+                        ""version"": ""12.0.3"",
+                        ""description"": ""Json.NET is a popular high-performance JSON framework for .NET"",
+                        ""summary"": """",
+                        ""title"": ""Json.NET"",
+                        ""iconUrl"": ""https://api.nuget.org/v3-flatcontainer/newtonsoft.json/12.0.3/icon"",
+                        ""licenseUrl"": ""https://www.nuget.org/packages/Newtonsoft.Json/12.0.3/license"",
+                        ""projectUrl"": ""https://www.newtonsoft.com/json"",
+
+                        ""tags"": [
+                            ""json""
+                        ],
+
+                        ""authors"": [
+                        ""James Newton-King""
+                        ],
+
+                        ""totalDownloads"": 531607259,
+                        ""verified"": true,
+
+                        ""packageTypes"": [
+                        {{
+                            ""name"": ""Dependency""
+                        }}
+                        ],
+
+                        ""versions"": [
+                        {{
+                            ""version"": ""3.5.8"",
+                            ""downloads"": 461992,
+                            ""@id"": ""https://api.nuget.org/v3/registration5-semver1/newtonsoft.json/3.5.8.json""
+                        }}
+                        ]
+                    }}
+                    ]
+                }}";
+
+                server.Get.Add("/search/query?q=logging&skip=0&take=20&prerelease=true&semVerLevel=2.0.0", r => queryResult);
+
+
+                server.Start();
+
+                // Act
+                var args = new[]
+                {
+                    "search",
+                    "logging",
+                };
+
+                var result = CommandRunner.Run(
```

Comment:
Explicit type instead of var.
Also, please use named parameters for all of the args here.